docs/tutorials/start-a-private-network/3keygen.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Generate Your Own Keys
+---
+
+Now that we know the fundamentals and the command line options, it's time to generate our own keys rather than using the well-known Alice and Bob keys. Each person who wants to participate in the blockchain should generate their own keys. This page explains several options for generating keys, and each participant only needs to choose one. Regardless of which option you choose, be sure to record all of the output from this section as you will need it later.
+
+## Option 1: Subkey
+Subkey is a tool the generates keys specifically designed to be used with Substrate. This is the most secure way to generate keys, and is how you should generate keys if you plan to use them in production.
+
+Begin by compiling and install the utility.
+```bash
+cargo install --force subkey --git https://github.com/paritytech/substrate
+```
+
+Generate a mnemonic and see the `sr25519` key and address associated with it. This key will be used by Aura for block production.
```

Comment:
It's better if there's a link to Aura's detail description.

Reply:
I agree I'll draft a PR for those write-ups so we can link to the. I don't want that to block this PR though.